traitlets/config/application.py

```diff
@@ -27,7 +27,9 @@
 from ipython_genutils.importstring import import_item
 from ipython_genutils.text import indent, wrap_paragraphs, dedent
 from ipython_genutils import py3compat
-from ipython_genutils.py3compat import string_types, iteritems
+
+import six
+from six import iteritems
```

Comment:
I would actually just get rid of all of our uses of iteritems. I don't think we ever use it in a case where it's beneficial.